### PR TITLE
Added usual MacOS default menu

### PR DESCRIPTION
**Pull request by Eskaan**
New Features:
 - option to open About Page
 - Services Submenu
 - Hide Others
 - Show All

## Purpose
The Mac OS Application Menu has usual a few more options than only to Hide and to Quit the App, like the About window, who shows the Author and the Version of the app and the "Services" submenu to track the task on the Taskmanager (Activity Monitor) and other things. Hope it will be accepted ;) but if not it wasnt that much work. :)
I also added a few pictures to compare before and after:
BeforeAfter
The "Electron" Name of the Menu results on the Developing envoirment, will change to the program name when launched from the app.

#### Open Questions and Pre-Merge TODOs
- [ ] Please **Check the Code** before merging, im not very used to node.js and Electron.

Hope it will be accepted ;) but if not it wasnt that much work. :)
LG Code-Ac aka Acrafts#7503 (on Discord)

### CODE REVIEW

File: public/electron.js [hunk 1]
```diff
@@ -75,15 +75,35 @@ const edit = [
...
+              role: 'hideOthers'
+            },
+            {
+              label: 'Show All',
+              role: 'unhide'
             },
             { type: 'separator' },
             {
-              label: 'Quit',
+              label: `Quit GDLauncher`,
```
Comment (blarfoon) [inline public/electron.js]: No need for the interpolation quote here, just use a single quote

File: public/electron.js [hunk 2]
```diff
@@ -75,15 +75,35 @@ const edit = [
...
               accelerator: 'Command+Q',
-              click: () => {
+              click() {
```
Comment (blarfoon) [inline public/electron.js]: Why changed this? If there's no need, leave it as it was

**Review by blarfoon — CHANGES_REQUESTED** [review]

File: public/electron.js (hunk 1 above)
Comment (Eskaan) [reply public/electron.js]: Oh, sorry, forgot to press the alt key on the keyboard while writing the quote 👍 I’ll fix that later (if you want you should be able too)

File: public/electron.js (hunk 2 above)
Comment (Eskaan) [reply public/electron.js]: Okay, reverted it.

File: public/electron.js (hunk 1 above)
Comment (Eskaan) [reply public/electron.js]: Actualy happend three times :/

**Comment by blarfoon** [comment]
Code looks alright, can anyone on mac test this?

**Review by blarfoon — APPROVED** [review]

**Pull request merged**